.catalog-onboard-pipeline.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: v1
+offerings:
+  - name: deploy-arch-ibm-monitoring-agent
+    kind: solution
+    catalog_id: 7df1e4ca-d54c-4fd0-82ce-3d13247308cd
+    offering_id: b0a00765-ee91-4c2a-87fc-df46dbc58cdb
+    variations:
+      - name: fully-configurable
+        mark_ready: true
+        install_type: fullstack
+        pre_validation: "tests/scripts/pre-validation-deploy-slz-roks-and-obs-instances.sh"
+        post_validation: "tests/scripts/post-validation-deploy-slz-roks-and-obs-instances.sh"
```

Comment:
Can you rename these scripts? Your are not even deploying SLZ in them since your using the base OCP module instead